src/test/scala/uk/gov/hmrc/api/helpers/IdentifyGroupHelper.scala

```diff
@@ -27,18 +27,23 @@ import java.nio.charset.StandardCharsets
 
 class IdentifyGroupHelper {
```

Comment:
This isn't a good name. I would suggest HttpClientHelper or something similar

Reply:
As discussed there are many classes that can be refactored later as this is not related to 1362